storybook/manager-head.html

```diff
@@ -1,10 +1,19 @@
 <script>
 	( function redirectIfStoryMoved() {
+		const PREVIOUSLY_EXPERIMENTAL_COMPONENTS = [ 'navigation' ];
```

Comment:
We can add components to this list as we remove their experimental status (#59418), and once all of them are done we can simplify to a bulk redirect on `components-experimental-` prefixes.

More info on the redirect mechanism at #59181.